debian/rules

```diff
@@ -65,7 +66,6 @@ CONFFLAGS_deb += \
 	--enable-coredump \
 	--enable-elfutils \
 	--disable-apparmor \
-	--enable-libcurl \
 	--enable-importd \
```

Comment:
Updated the PR removing curl from the build deps. I was aware we were disabling importd, but it shouldn't be a problem since we don't use it on EOS.